force-app/main/default/lwc/tableWithAccounts/tableWithAccounts.js

```diff
@@ -1,57 +1,58 @@
 import { LightningElement, track } from 'lwc';
 import getDetails from '@salesforce/apex/AccountIntegrationController.getDetails';
+import patchDetails from '@salesforce/apex/AccountIntegrationController.patchDetails';
 import returnAccounts from '@salesforce/apex/AccountIntegrationController.returnAccounts';
+import updateReturnAccounts from '@salesforce/apex/AccountIntegrationController.updateReturnAccounts';
 import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import { refreshApex } from '@salesforce/apex';
 
 const pageSize = 7;   
 const columns = [
+    //add colum org
     {label: 'Name', fieldName : 'name'},
     {label: 'Account Number', fieldName : 'accountNumber', editable : true},
     {label: 'Ticker Symbol', fieldName : 'tickerSymbol', editable : true},
     {label: 'Rating', fieldName : 'rating', editable : true},
-    {label: 'Type', fieldName : 'type', editable : true}
+    {label: 'Type', fieldName : 'type', editable : true},
+    {label: 'Org', fieldName : 'org'}
 ];
-
 export default class TableWithAccounts extends LightningElement {
 
     @track listAcc = [];
     @track isLoading = false;
     @track page = 1;
     @track totalPage;
+    @track draftValues = [];
 
-    detailsPromise = [];
-    accountsPromise = [];
+    returnAccIds = [];
+    detailsIds = [];
 
     combinedAccounts = [];
     startingRecord = 1;
     endingRecord = 0;
     totalRecordCount;
     columns = columns;
 
-    fildsItemValues = [];
-
-
 
-
-    connectedCallback() {
+    connectedCallback() { 
         this.loadingAccounts();
     }
 
     loadingAccounts() {
         this.isLoading = true;
+        
 
-        this.accountsPromise = returnAccounts();
-        this.detailsPromise = getDetails();
-        console.log('result 1+++' + this.detailsPromise);
-        console.log('result 1+++' + this.accountsPromise);
+        this.returnAccIds = returnAccounts();
```

Comment:
this is asynchronous function here is just calling it. It is just placed into the queue. It means the next rows will be executed immediately and you can't be sure accounts were returned when this function is finished.